core/vm/logger.go

```diff
@@ -65,36 +58,34 @@ type StructLog struct {
 	Err     error
 }
 
+type VmLogger interface {
+	CaptureState(env Environment, pc uint64, op OpCode, gas, cost *big.Int, memory *Memory, stack *stack, contract *Contract, depth int, err error);
+}
```

Comment:
Better names would be Logger (sadly unavailable a.t.m,), StructLogger or StateCapturer.
The "vm" part is implicit in the package name.